ext/js/templates/sandbox/anki-template-renderer.js

```diff
@@ -531,6 +534,7 @@ export class AnkiTemplateRenderer {
     _concat(args) {
         let result = '';
         for (let i = 0, ii = args.length; i < ii; ++i) {
+            // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
```

Comment:
I'm surprised this is needed, isn't everything a string here?

Reply:
It's needed because `args` is `unknown[]`. I would have made it so that this disable wasn't necessary, but there's no way to specifically allow `unknown` stringification without disabling the rule, and I think this was the only place that had an issue, so I just suppressed this location.